### ISSUE TITLE
Update ignite to the latest version

### ISSUE THREAD

**Issue by sveneberth** [viur-framework/viur-base#48]
Although the ignite submodule was updated by https://github.com/viur-framework/viur-base/pull/36 to almost the latest version, the corresponding changes were not made.
Some structural changes are necessary and the base needs to initialize the ignite elements.

**Comment by phorward** [comment]
Is this done by #53 now?

**Comment by sveneberth** [comment]
> Is this done by #53 now?

Nope. The project LESS and HTML must be changes. There're several (breaking) [changes](https://github.com/viur-framework/viur-ignite/blob/master/CHANGELOG.md#400---2021-02-26). In ignite there is a [backward compatibility](https://github.com/viur-framework/viur-ignite/blob/4.0.0/foundation/deprecated.less) for the element initialization, but I'm not sure if this works for all changes and should be updated in the future anyway... AFAIK some CSS classes or the structure (HTML) how to nest them were changed and needs to be adjusted (a few changes were also needed in the [docs](https://github.com/viur-framework/ignite-docs/compare/ca7e4c6b3c8223fadd337af49ee174fbe26b96bf...master)).

**Comment by phneutral** [comment]
I'll look at the whole issue (hopefully) over the next week.

**Comment by phorward** [comment]
@phneutral @sveneberth bump! Can we merge this with https://github.com/viur-framework/viur-base/tree/rework asap?
